Reviews/219 - Casper Analytics - iOS-Android-Web/5 - Integrate Casper Signer/README.md

```diff
@@ -1,314 +1,10 @@
-    117|         }
-    118|
-  > 119|         throw new ExpectationFailedException(
-    120|             $failureDescription,
-    121|             $comparisonFailure
-    122|         );
-    123|     }
-
-  1   C:\Users\EKON3\source\VsRepos\casper-analytics-api\vendor\phpunit\phpunit\src\Framework\Constraint\IsIdentical.php:102
-      PHPUnit\Framework\Constraint\Constraint::fail("")
-
-  2   C:\Users\EKON3\source\VsRepos\casper-analytics-api\vendor\phpunit\phpunit\src\Framework\Assert.php:2924
-      PHPUnit\Framework\Constraint\IsIdentical::evaluate("")
-```
-
-
-
-Requirement | Finding
------------- | -------------
-Unit Tests - At least one positive path test | PASS
-Unit Tests - At least one negative path test | PASS
-Unit Tests - Additional path tests | PASS
-
-# Documentation
-
-### Code Documentation
-
-Reviewer observed that all critical functions of the project had low-level code documentation which could be used for automated documentation generation.
-
-Requirement | Finding
------------- | -------------
-Code Documented | PASS
-
-### Project Documentation
-
-The reviewer observed that the README of the project has detailed general and usage documentation.
-
-Requirement | Finding
------------- | -------------
-Usage Documented | PASS
-
-## Overall Conclusion on Documentation
-
-The reviewer concludes that the project has sufficient comprehensive general documentation. 
-
-# Open Source Practices
-
-## Licenses
-
-The Project is released under the MIT License
-
-Requirement | Finding
------------- | -------------
-OSI-approved open source software license | PASS
-
-## Contribution Policies
-
-Pull requests and Issues are enabled. And, the project contain a CONTRIBUTING policy. 
-
-Requirement | Finding
------------- | -------------
-OSS contribution best practices | PASS
-
-# Coding Standards
-
-## General Observations
-
-Code is generally well-structured and readable. The Github linked project is easy and successful to install, but an error was encountered during the installation phase of the app project. An error was received in the unit tests of the project. Manual tests fail for making a delegation using the casper signer. These errors prevent the project from succeeding.
-
-# Final Conclusion
-The project provides the most of the functionalities described in the grant application and milestone acceptance criteria. 
-The reviewer praises the OP for the general stability and the fluid user experience of the app.
-However, it is recommended to fix test errors in the lib file. Also the "delegate using casper signer" in the milestone doesn't work properly. 
-Thus, in the reviewer's opinion, this submission should fail.
-
-# Recommendation
-
-Recommendation | FAIL
------------- | -------------
-
+Grant Proposal | [219 - Casper Analytics - iOS/Android/Web](https://portal.devxdao.com/public-proposals/219)
+------------ | -------------
+Milestone | 5
+Milestone Title | Integrate Casper Signer
+OP | tmesser

```

Comment:
I don't think this is the right OP.

Reply:
I've edited OP. Please let me know if you want any other changes to the table in the readme file.